src/pages/courseDetailsPage/CourseDetailsPage.tsx

```diff
@@ -5,31 +5,14 @@ import {
   ArrowIcon,
   BackLink,
   CourseDescription,
-  CourseLesson,
-  CourseLessonsList,
-  LessonTitle,
   CourseTitle,
-  LockIcon,
-  Title,
-  TitleBox,
   VideoContainer,
-  CurrentLesson,
-  CurrentLessContainer,
-  CurrentText,
 } from "./CourseDetailsPage.styled";
 import { Container } from "components/CoursesList/CoursesList.styled";
```

Comment:
You have broken imports in your project:
<img width="327" alt="image" src="https://github.com/Genesis-Front-End-School/clean-code-principes-yulyaolshanska/assets/18702961/3e90f2a9-58e8-4c45-aaf6-11cfafba9ad8">

``CoursesLists`` folder was renamed to `coursesLists`, but imports were not resolved correctly.

Reply:
<img width="517" alt="image" src="https://github.com/Genesis-Front-End-School/clean-code-principes-yulyaolshanska/assets/63751946/698a7edf-8e2f-44c3-a884-79c5060aa8d4">
<img width="785" alt="image" src="https://github.com/Genesis-Front-End-School/clean-code-principes-yulyaolshanska/assets/63751946/49b164d0-b912-475e-91d1-bd28a603d3ce">